src/training_data.py

```diff
@@ -77,16 +77,16 @@ def load_luis_data(self,filename):
         warnings.warn(
         """LUIS data may not always be correctly imported because entity locations are specified by tokens.
         If you use a tokenizer which behaves differently from LUIS's your entities might not be correct""")
-        data = json.loads(open(filename,'rb').read())
+        data = json.loads(codecs.open(filename, encoding='utf-8').read())
         for s in data["utterances"]:
-            text = unicode(s.get("text"))
-            tokens = [t.decode('utf-8') for t in self.tokenizer.tokenize(text)]
+            text = s.get("text")
+            tokens = [t for t in self.tokenizer.tokenize(text)]
```

Comment:
do the tests cover the case where tokenizer is a MITIETokenizer? (just to be sure this works in all cases)